sormas-api/src/main/java/de/symeda/sormas/api/statistics/StatisticsCaseAttribute.java

```diff
@@ -45,6 +45,18 @@
 		StatisticsCaseSubAttribute.EPI_WEEK_OF_YEAR,
 		StatisticsCaseSubAttribute.DATE_RANGE),
 
+	OUTCOME_TIME(StatisticsCaseAttributeGroup.TIME,
+		false,
+		false,
```

Comment:
please set "unknownValueAllowed" to True otherwise the report will not work on column selection